.circleci/cimodel/data/pytorch_build_data.py

```diff
@@ -59,7 +59,7 @@
             ]),
             ("11.1", [
                 ("3.8", [
-                    X(True),
+                    ("important", [X(True)]),
```

Comment:
Turning this on for this PR so I could see the result of my code to better debug.